RateLimiter/RateLimiter.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2;net472</TargetFrameworks>
+    <TargetFrameworks>netstandard2.1;net472</TargetFrameworks>
```

Comment:
The library already supports netstandard2.1 as it supports netstandard2.0. 

This change would make the library ONLY work on netstandard2.1.